torch/csrc/jit/python/init.cpp

```diff
@@ -1243,9 +1244,12 @@ void initJITBindings(PyObject* module) {
     }
   });
 
-  m.def("wait", [](const std::shared_ptr<PythonFutureWrapper>& fut) {
-    return fut->wait();
-  });
+  m.def(
+      "wait",
+      [](const std::shared_ptr<PythonFutureWrapper>& fut,
+         bool non_blocking = false) { return fut->wait(non_blocking); },
+      py::arg("future"),
+      py::arg("non_blocking") = false);
```

Comment:
Looks like we never added docs for this method, we should add docs and explain the non_blocking flag. In particular we should mention that this flag is a no-op for the CPU future.